composer.json

```diff
@@ -26,6 +26,7 @@
         "drupal/admin_toolbar": "^2.0",
         "drupal/advagg": "^4.0",
         "drupal/better_exposed_filters": "^4.0",
+        "drupal/captcha": "^1.1",
```

Comment:
W8, ale czemu dodajemy captcha do Drooplera, a nie do projektu na którym stoi demo? IMO to powinno być w repo tej konkretnej strony która tego potrzebuje